packages/appium/test/cli-e2e-specs.js

```diff
@@ -4,10 +4,14 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { tempDir, fs, mkdirp, util } from 'appium-support';
 import { KNOWN_DRIVERS } from '../lib/drivers';
+import { PROJECT_ROOT as cwd } from './helpers';
 
 chai.should();
 chai.use(chaiAsPromised);
-const cwd = path.resolve(__dirname, '..', '..');
+
+// cannot use `require.resolve()` here (w/o acrobatics) due to the ESM context.
```

Comment:
acrobatics 😄

Reply:
not sure if it's even possible with babel.  babel lets you do things you couldn't otherwise, but I don't think it supports all of Node.js' native ESM handling